README.md

```diff
@@ -1,24 +1,144 @@
-# README
+## Rales Engine Project
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+### Description
 
-Things you may want to cover:
+Nick and Kyle's [Rales Engine](http://backend.turing.io/module3/projects/rails_engine) project for Turing. In this project we import business data and host an API with endpoints to return records, relationships between models, and business intelligence as JSON. Checkout the schema file `schema.png` in the public directory to see the models and their relationshps.
```

Comment:
What would you think about linking to the GitHub repo for the project instead of the curriculum? (This is the link: https://github.com/turingschool/lesson_plans/blob/master/ruby_03-professional_rails_applications/rails_engine.md)

Let's tell the user explicitly how to view the schema (and I love that you included that - great idea).  Is it at: 
`http://localhost:3000/public/schema.png` ?